kingpin/actors/aws/ecs.py

```diff
@@ -543,6 +540,9 @@ class Service(ECSBaseActor):
 
     **Options**
 
+    :state:
+      Present or Absent. Defaults to Present.
```

Comment:
Use the same string here as below, "Desired state: present/absent. Defaults to present."